buildkite/README.md

```diff
@@ -244,6 +244,12 @@ platforms:
 [...]
 ```
 In this example the jobs on Windows and MacOS would use 0.20.0, whereas the job on Ubuntu would run 0.18.0.
+
+You can also use unreleased versions of Bazel:
+
+- `last_green` refers to the Bazel binary built at the latest green commit in the [Bazel pipeline](https://buildkite.com/bazel/bazel-bazel).
+- `commit` tells the pipeline to build a new Bazel binary at the current commit and to use that binary for all other Bazel-related steps (clean, run, build and test).
```

Comment:
"The current commit" sounds like HEAD?

Reply:
It's HEAD most of the time. However, Buildkite allows users to specify the commit at which they want to build, which can be different from HEAD.